jaseci_ai_kit/jaseci_ai_kit/modules/ph/requirements.txt

```diff
@@ -0,0 +1,4 @@
+torchvision
```

Comment:
Do we actually need torchvision and torchaudio? Just trying to make sure we don't blow up the dependency list of jaseci_ai_kit. Also, do we need pytorch-cuda? 

If these are must-haves, can you add them to the setup.py under jaseci_ai_kit? so they will get installed when you run pip install jaseci_ai_kit

Reply:
they are not needed, removed them